buildSrc/src/main/kotlin/com/saveourtool/save/buildutils/kotlin-library.gradle.kts

```diff
@@ -58,55 +48,20 @@ kotlin {
             }
     }
 
-    /*
-     * Common structure for MPP libraries:
```

Comment:
by the way, why its so easily removed now? and how nothing broke?

Reply:
it's removed since: https://youtrack.jetbrains.com/issue/KT-58676/Enable-default-Kotlin-Target-Hierarchy-by-default
Kotlin suggests to not use dependsOn explicitly.

It does work because we move all from `commonNonJsMain` to `commonMain` and disabled tests for JS.

Created #557 to resolve this issue fully